grype/grypeerr/errors.go

```diff
@@ -0,0 +1,6 @@
+package grypeerr
+
+var (
+	// ErrAboveSeverityThreshold indicates when a vulnerability severity is discovered that is above the given --fail-on severity value
+	ErrAboveSeverityThreshold = NewExpectedErr("discovered vulnerabilities at or above the severity threshold")
```

Comment:
I would probably move specific errors out of the general error package and closer to where the error is relevant. For example, if the severity checking logic is in the `cmd` package right now, that's where I'd put this error.

Reply:
Agreed, I'll move this over to `cmd`

Reply:
per our conversation, let's iterate on this --we should try not to have a global sentinel errors package, but probably isn't a show stopper for now (especially since we don't have a better pattern off hand yet)